src/components/ListItem.jsx

```diff
@@ -11,6 +13,25 @@ export function ListItem({ listPath, item, name }) {
 		setIsChecked(!isChecked);
 	}
 
+	async function handleDelete() {
+		console.log('list path', listPath);
+		console.log('item', item);
+
+		if (window.confirm('Are you sure you want to delete this item?') === true) {
```

Comment:
Not sure we need the `=== true` part here, since confirm() returns a boolean anyway.

Reply:
I believe that it was necessary for the confirm, but I will double check!

Reply:
removed!